gsa/src/web/pages/tasks/row.js

```diff
@@ -110,11 +110,11 @@ const Row = ({
     if (hasValue(observers.users)) {
       obs.user = _('Users {{user}}', {user: observers.users.join(', ')});
     }
-    if (observers.roles.length > 0) {
+    if (observers?.roles?.length > 0) {
```

Comment:
Would be better to use
```suggestion
    if (isDefined(observers?.roles?.length) && observers.roles.length > 0) {
```
to not rely on js' behavior of comparing undefined with 0.